docs/source/enable_cc_lifecycle.md

```diff
@@ -240,9 +240,9 @@ For this channel update to be approved, the policy for modifying the `Channel/Ap
 
 ## Enable new lifecycle in `core.yaml`
 
-If you follow [the recommended process](./upgrading_your_components.html#overview) for using a tool like `diff` to compare the new version of `core.yaml` packaged with the binaries with your old one, you will not need to whitelist `_lifecycle: enable` because the new `core.yaml` has added it under `chaincode/system`.
+If you follow [the recommended process](./upgrading_your_components.html#overview) for using a tool like `diff` to compare the new version of `core.yaml` packaged with the binaries with your old one, you will not need to allowlist `_lifecycle: enable` because the new `core.yaml` has added it under `chaincode/system`.
 
-However, if you are updating your old node YAML file directly, you will have to add `_lifecycle: enable` to the system chaincodes whitelist.
+However, if you are updating your old node YAML file directly, you will have to add `_lifecycle: enable` to the system chaincodes allowlist.
```

Comment:
```suggestion
However, if you are updating your old node YAML file directly, you will have to add `_lifecycle: enable` to the list of enabled system chaincodes.
```

Reply:
I'm making similar change in core.yaml comments:
https://github.com/hyperledger/fabric/pull/1461